spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 
 # Requires supporting ruby files with custom matchers and macros, etc,
 # in spec/support/ and its subdirectories.
-Dir[Rails.root.join('spec/support/**/*.rb')].sort.each { |f| require f }
+Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }
```

Comment:
Rubocop: Ruby 3 performs this sort by default. Nicely slaying a whole suite of bugs that manifest differently on the CI vs development.